.github/workflows/jacoco.yml

```diff
@@ -0,0 +1,111 @@
+#
+# Copyright 2022 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Build
+
+on:
+  push:
+    branches: [master ]
+  pull_request:
+      types: [ assigned, opened, synchronize, reopened, labeled ]
+
+jobs:
+  jacoco:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:15
+        env:
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: movies
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 5432:5432
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - scala: 2.13.12
+            scalaShort: "2.13"
+            overall: 75.0
+            changed: 80.0
```

Comment:
again, this is a single run

Reply:
I want to keep these "variables" in the same place for easy future updates. (Better than looking inside the code and forgetting to change "2.13" to "2.14" deep in paths)

Reply:
Why not use ENV variables?

Reply:
Addressed.